packages/blocks/image/src/components/Media.tsx

```diff
@@ -272,9 +273,8 @@ export const Media: BlockComponent<
                   .filter(
                     (
                       link,
-                    ): link is BlockProtocolLink & SingleTargetLinkFields => {
-                      return "linkId" in link;
-                    },
+                    ): link is BlockProtocolLink & SingleTargetLinkFields =>
+                      isSingleTargetLink(link),
```

Comment:
The only issue with `filter(isSingleTargetLink)` is that `filter` passes multiple arguments, so in theory if `isSingleTargetLink` was updated to accept multiple arguments, it would start receiving the ones that `filter` sends automatically, which might be undesired.

In this case it seems that `isSingleTargetLink` will never accept multiple arguments since it is meant to be a typeguard, but something to bear in mind when deciding between implicit and explicit argument passing.

Reply:
> `(link) => isSingleTargetLink(link)`

This doesn't work, it needs `link is BlockProtocolLink & SingleTargetLinkFields` to be applied to the return type of the function to satisfy the TS compiler.

> `.filter(isSingleTargetLink)`

However, this does work.